src/util/envVariablesValidations.ts

```diff
@@ -15,13 +15,19 @@
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
 const expectedVariables = [
-  'REACT_APP_PAIR',
   'REACT_APP_SUBSTRATE_PROVIDER_CHAIN_1',
   'REACT_APP_SUBSTRATE_PROVIDER_CHAIN_2',
   'REACT_APP_LANE_ID',
   'REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS'
 ];
 
+const optionalVariables = [
+  'REACT_APP_CUSTOM_TYPES_URL_CHAIN_1',
+  'REACT_APP_CUSTOM_TYPES_URL_CHAIN_2',
+  'REACT_APP_CUSTOM_HASHER_CHAIN_1',
+  'REACT_APP_CUSTOM_HASHER_CHAIN_2'
```

Comment:
Cool stuff. Wdyt about chaing the variable names though to have `CHAIN_1` in front, so that you can easily sort them:
```
REACT_APP_CHAIN_1_CUSTOM_TYPES_URL_=https://raw.githubusercontent.com/paritytech/parity-bridges-common/master/deployments/types-rialto.json
REACT_APP_CHAIN_1_SUBSTRATE_PROVIDER=wss://wss.rialto.brucke.link
REACT_APP_CHAIN_2_CUSTOM_TYPES_URL=https://raw.githubusercontent.com/paritytech/parity-bridges-common/master/deployments/types-millau.json
REACT_APP_CHAIN_2_SUBSTRATE_PROVIDER=wss://wss.millau.brucke.link
REACT_APP_CHAIN_2_CUSTOM_HASHER_=blake2Keccak256Hasher

REACT_APP_LANE_ID=0x00000000
REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS=false
```
The reason for that is that I think it's easier to think "per chain" instead of "per functionality". Like when you start defining your env variables, you will most likely go with:
1. What are the things for one chain I need to configure?
2. What are the thigns for second chain I need to configure?

Instead of:
1. What are the custom types for chain1 and chain2?
2. What are the custom hashers for chain1 and chain2?
3. What are the URLs for chain1 and chain2?

Basically - it's easier to enumerate chains than it is to enumerate all possible configuration options from a developer perspective that is configuring the UI.

Reply:
Yeap. Sounds good to me